src/components/Dapp.re

```diff
@@ -102,7 +94,9 @@ module AuctionDisplay = {
     let optCurrentUsdEthPrice = UsdPriceProvider.useUsdPrice();
 
     <>
-      <h3> "Auction"->React.string </h3>
+      <ActionButtons.Auction animal />
+      <br />
+      <br />
```

Comment:
I put the "action" buttons at the top all in a line (rather than below the text). Looks better and more uniform IMO...